backend/app/agents/devrel/github/services/github_mcp_server.py

```
@@ -0,0 +1,91 @@
+async def health_check():
+    """Health check endpoint"""
+    return {"status": "healthy", "service": "github-mcp"}
+
+class OrgInfoRequest(BaseModel):
+    org: str
+
+@app.post("/list_org_repos")
+async def list_org_repos(request: OrgInfoRequest):
+    try:
+        if not github_service:
+            raise HTTPException(status_code=500, detail="GitHub service not available")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)